railties/lib/rails/commands/server.rb

```diff
@@ -94,7 +94,8 @@ def default_options
         environment:        (ENV['RAILS_ENV'] || ENV['RACK_ENV'] || "development").dup,
         daemonize:          false,
         caching:            nil,
-        pid:                Options::DEFAULT_PID_PATH
+        pid:                Options::DEFAULT_PID_PATH,
+        restart_cmd:        'bin/rails s'
```

Comment:
Question: if I start the rails server using, say:

    rails server -b 0.0.0.0 -p 4567

And then restart the server using:

    rails touch

Will the same options be used?